tutorials/quick_start/keras_fw/quant.py

```diff
@@ -40,15 +43,35 @@ def get_tpc(target_platform_name: str, target_platform_version: str) -> TargetPl
     return mct.get_target_platform_capabilities('tensorflow', target_platform_name, target_platform_version)
 
 
+def get_target_kpi(model, weights_compression, representative_data_gen, core_config, tpc):
+    """
+    Calculates the model's required size according to the given weights compression rate, to provide as a constraint for mixed precision search.
+
+    Args:
+        model: The model to calculate the KPI.
+        weights_compression: The required weights compression ratio.
+        representative_data_gen: Callable function to generate the representative dataset.
+        core_config (CoreConfig): CoreConfig containing parameters for quantization and mixed precision.
+        tpc (TargetPlatformCapabilities): TargetPlatformCapabilities to optimize the TensorFlow model according to.
+
+    Returns:
+        A KPI object computed from MCT and contains info about the target model size.
+
+    """
+    kpi_data = mct.core.keras_kpi_data_experimental(model, representative_data_gen, core_config=core_config, target_platform_capabilities=tpc)
+    weights_kpi = BYTES_TO_FP32 * kpi_data.weights_memory / weights_compression # (4 bytes for fp32) * weights memory(in Bytes) / compression rate
+    return KPI(weights_kpi)
```

Comment:
you might want to do:
`KPI(weights_memory=weights_kpi)`
since KPI object gets other arguments as well, and in the future you might want to pass them as well